.travis.yml

```diff
@@ -12,7 +12,7 @@ git:
 language: java
 install: true
 jdk:
- - openjdk8
+ - openjdk11
```

Comment:
The client must run well also on jdk8 how can we test it?

Reply:
Can you clarify a little bit? Would relying on the unit/integration tests be insufficient?

Reply:
I can't see here where we are testing with jdk8

Reply:
I believe we can set the `-source` and `-target` javac flags for the client which should preserve compatibility,

Reply:
I have kept all the system test pod defaults as is, so we can still rely on running those for some level of verification. I'm not sure if there is a standard convention (for the variable name) but now if a user has a `JAVA_8` env variable defined, it will attempt to run the unit tests using the executable that `JAVA_8` points to.

Reply:
@tkaitchuck  The best way is to set "-release 8" (as it is the current approach), otherwise javac will pick wrong overloaded versions in newer JDKs.